src/libAtomVM/context.c

```diff
@@ -270,9 +297,23 @@ bool context_get_process_info(Context *ctx, term *out, term atom_key)
             break;
         }
 
+        // pids of linked processes
+        case LINKS_ATOM: {
+            term_put_tuple_element(ret, 0, LINKS_ATOM);
+            term list = term_nil();
+            struct ListHead *item;
+            LIST_FOR_EACH (item, &ctx->monitors_head) {
+                struct Monitor *monitor = GET_LIST_ENTRY(item, struct Monitor, monitor_list_head);
+                if (monitor->ref_ticks == 0) {
```

Comment:
Why this condition?  (Should we have a comment?)

Reply:
It means links (as opposed to monitors). This semantic was added quite some time ago.
I've been happy to add a comment. Plus it may be obvious if one day we implement `monitors` key to `process_info/2`.

https://github.com/atomvm/AtomVM/compare/c4a753f38e8a9d137bbd238014637b8e0c8e3673..4b87d20235a64295f649e2751cfecdfc1b0247eb